packages/nextjs/src/config/withSentryConfig.ts

```diff
@@ -67,6 +68,27 @@ function getFinalConfigObject(
     }
   }
 
+  if (process.argv.includes('--experimental-build-mode')) {
+    if (!showedExperimentalBuildModeWarning) {
+      showedExperimentalBuildModeWarning = true;
+      // eslint-disable-next-line no-console
+      console.warn(
+        '[@sentry/nextjs] The Sentry Next.js SDK does not currently fully support next build --experimental-build-mode',
```

Comment:
I would like us to expand this comment to be more specific to what we don't support, or link to a docs page that explains this in more detail.